README.md

```diff
@@ -16,15 +16,19 @@ The different fields include:
  - IO data (bytes in and out) per interval
  - global amount of messages in queues on a node
 
-## How to build ##
+## How to build
 
- `$ ./rebar compile`
+````sh
+$ rebar get-deps compile
+```
 
 or
 
- `$ make`
+```sh
+$ make
+```
 
-## Why scheduler wall time statistics disabled by default? ##
+## Why scheduler wall time statistics disabled by default?
 
 Scheduler wall time statistics are now disabled by default to keep in line with 0.1.0 behaviour, and after some bugs being reported when the Erlang scheduler would lock on such calls in R15B01 a few times in a day, never to unlock again. People who want to take the risk of running these statistics can do it by setting the `vmstats` env variable `sched_time` to `true`.
 
```

Comment:
We could maybe enable the scheduler by default and deprecate R15 if we're moving to 2.0.0. What do you think?